doc/computation.rst

```diff
@@ -70,10 +71,25 @@ for working with missing data from pandas:
     x.count()
     x.dropna(dim='x')
     x.fillna(-1)
+    x.ffill()
+    x.bfill()
 
 Like pandas, xarray uses the float value ``np.nan`` (not-a-number) to represent
 missing values.
 
+xarray objects also have an :py:meth:`~xarray.DataArray.interpolate_na` method
+for filling missing values via 1D interpolation.
+
+.. ipython:: python
+
+    x = xr.DataArray([0, 1, np.nan, np.nan, 2], dims=['x'],
+                     coords={'xx': [0, 1, 1.1, 1.9, 3]})
```

Comment:
I think here is a typo. Maybe
```python
x = xr.DataArray([0, 1, np.nan, np.nan, 2], dims=['x'],
                    coords={'xx': ('x', [0, 1, 1.1, 1.9, 3])})
```
or
```python
x = xr.DataArray([0, 1, np.nan, np.nan, 2], dims=['x'],
          coords={'x': [0, 1, 1.1, 1.9, 3]})
```
?

Reply:
This was intentional.

Reply:
Really?
I got
`ValueError: coordinate xx has dimensions ('xx',), but these are not a subset of the DataArray dimensions ['x']`
from this script.

Reply:
You were right. I was missing the Variable constructor.